examples/taskworkflow.md

```diff
@@ -0,0 +1,87 @@
+### About
+The `taskworkfow.go` file contains code to pull and unpack a container image
+using the `naive` snaphotter, create a container and execute a task using the
+`firecracker-containerd` runtime.
+
+### Building
+After checking out the `firecracker-containerd` repo, you can build this
+example using the `go build` command. Example:
+```bash
+$ go build -o /tmp/taskworkflow examples/taskworkflow.go
```

Comment:
Can you add this to the Makefile as well?  I think `make examples` would be good.  Then we can include compiling the examples in our CI so they don't break.